test/extended/oauth/http1.go

```diff
@@ -0,0 +1,44 @@
+package oauth
+
+import (
+	"encoding/json"
+	"net/http"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"golang.org/x/net/http2"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/origin/pkg/oauth/util"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("The OAuth server", func() {
```

Comment:
not sure this is the right name format

Reply:
(discussed in-person, will fix later as needed)